commitlint.config.js

```diff
@@ -2,6 +2,6 @@ module.exports = {
     extends: ['@commitlint/config-conventional'],
     rules: {
         'subject-case': [1, 'always', ['lower-case']],
-        'body-max-line-length': [0],
+        'footer-leading-blank': 'never'
```

Comment:
wait why did you remove the body max length rule?
also shouldn't we add a rule for footer max length too?

Reply:
The docs says the default for both is infinity